Source/ARTRealtime.m

```diff
@@ -641,8 +641,10 @@ - (void)onConnected:(ARTProtocolMessage *)message {
             [self.connection setMaxMessageSize:message.connectionDetails.maxMessageSize];
             if (!_resuming) {
                 [self.connection setSerial:message.connectionSerial];
```

Comment:
Why are you only setting the connectionSerial if not resuming? RTN10a isn't conditional

Reply:
Right, fixed 5b61635.